requirements.txt

```diff
@@ -1,238 +1,830 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
This is a pretty big diff that seems to be mostly adding line breaks... Is this an editor issue?

Reply:
so looks like this is caused by the version of `pip-tools`. This is older version format (I tried `pip-tools==5.4.0`), the newer versions use a new format. We should update this with another PR.